packages/components/src/FormField/FormFieldWrapper.tsx

```diff
@@ -124,97 +89,123 @@ export function FormFieldWrapper({
-                htmlFor={identifier}
-                style={
-                  prefixRef?.current || suffixRef?.current
-                    ? labelStyle
-                    : undefined
-                }
-              >
-                {placeholder}
-              </label>
-            )}
-
-            {prefix?.label && <AffixLabel {...prefix} labelRef={prefixRef} />}
-
-            <div className={styles.childrenWrapper} tabIndex={-1}>
-              {children}
-            </div>
-
-            {suffix?.label && (
-              <AffixLabel {...suffix} labelRef={suffixRef} variation="suffix" />
-            )}
-          </div>
-          {showClear && <ClearAction onClick={onClear} />}
-          {suffix?.icon && (
-            <AffixIcon {...suffix} variation="suffix" size={size} />
-          )}
-        </div>
-        <AnimatePresence
-          initial={toolbarVisibility === "always" ? false : true}
-        >
-          {isToolbarVisible && (
-            <motion.div
-              key="toolbar"
-              initial={toolbarAnimationEnd}
-              animate={toolbarAnimationStart}
-              exit={toolbarAnimationEnd}
-              transition={{
-                duration: tokens["timing-base"] / 1000,
-                ease: "easeInOut",
-              }}
-              tabIndex={-1}
-            >
-              <div
-                className={styles.toolbar}
-                data-testid="ATL-InputText-Toolbar"
-              >
-                {toolbar}
-              </div>
-            </motion.div>
-          )}
-        </AnimatePresence>
-      </div>
-      {description && !inline && (
-        <FormFieldDescription
-          id={descriptionIdentifier}
-          description={description}
+        <FormFieldInputHorizontalWrapper>
+          <AffixIcon {...prefix} size={size} />
+          <FormFieldInputWrapperStyles>
+            <FormFieldLabel
+              placeholder={placeholder}
+              identifier={identifier}
+              style={
+                prefixRef?.current || suffixRef?.current
+                  ? labelStyle
+                  : undefined
+              }
+            />
+            <AffixLabel {...prefix} labelRef={prefixRef} />
+
+            <FormFieldWrapperMain>{children}</FormFieldWrapperMain>
+
+            <AffixLabel {...suffix} labelRef={suffixRef} variation="suffix" />
+          </FormFieldInputWrapperStyles>
+          <ClearAction onClick={onClear} visible={showClear} />
+          <AffixIcon {...suffix} variation="suffix" size={size} />
+        </FormFieldInputHorizontalWrapper>
+        <FormFieldWrapperToolbar
+          toolbarVisibility={toolbarVisibility}
+          isToolbarVisible={isToolbarVisible}
+          toolbarAnimationEnd={toolbarAnimationEnd}
+          toolbarAnimationStart={toolbarAnimationStart}
+          toolbar={toolbar}
         />
-      )}
-      {error && !inline && <InputValidation message={error} />}
+      </div>
```

Comment:
A V2 comment: Is there an opportunity to get rid of these Divs and leverage our own components instead?